src/synth/fluid_voice.c

```diff
@@ -1113,7 +1113,7 @@ fluid_voice_update_param(fluid_voice_t* voice, int gen)
  */
 int fluid_voice_modulate(fluid_voice_t* voice, int cc, int ctrl)
 {
-  int i, k;
+  unsigned int i, k;
```

Comment:
IMHO, there's no reason to use unsigned integers here, except perhaps readability. But unsigned integers often causes unnecessary bugs [1], and at least when I see one, I check that the usage of it is not dangerous. By not using unsigned integers you both avoid these bugs and the need to check if the use of unsigned can cause bugs. IMO, unsigned integers should only be used when doing bit-operations, for instance to store flags, plus in rare situations where you need that extra bit and using a larger integer type is not a good option.

[1] Search the internet. For instance, Qt does not use unsigned integers, and the google style guide recommends not using them (at least that's what I've read, but I would be surprised if it didn't.).

Reply:
Here's what the google style guide says about integers: https://google.github.io/styleguide/cppguide.html#Integer_Types